x-pack/plugins/apm/public/components/app/RumDashboard/UserPercentile/index.tsx

```diff
@@ -44,8 +44,7 @@ export function UserPercentile() {
     if (!percentile) {
       updatePercentile(DEFAULT_P);
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  });
```

Comment:
It's ok that this hook evaluates on each render now?

Reply:
yes, it's totally fine.